.woodpecker/.phpunit.yml

```diff
@@ -67,6 +67,9 @@ steps:
       MEMCACHED_HOST: "memcached"
       MEMCACHE_HOST: "memcached"
     commands:
+      - if [ "${PHP_MAJOR_VERSION}" = "8.2" ]; then
```

Comment:
I'd prefer to add a new step "PHP Stan" and to use the `when`-block in woodpecker - see https://woodpecker-ci.org/docs/usage/workflow-syntax#when---conditional-execution

Reply:
Thanks, I moved phpstan into an own step.